examples/todos-with-undo/containers/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { ActionCreators } from 'redux-undo';
+import { addTodo, completeTodo, setVisibilityFilter, VisibilityFilters } from '../actions';
+import AddTodo from '../components/AddTodo';
+import TodoList from '../components/TodoList';
+import Footer from '../components/Footer';
+
+class App extends Component {
+  render() {
+    // Injected by connect() call:
+    const { dispatch, visibleTodos, visibilityFilter } = this.props;
+    return (
+      <div>
+        <AddTodo
+          onAddClick={text =>
+            dispatch(addTodo(text))
+          } />
+        <TodoList
+          todos={visibleTodos}
+          onTodoClick={index =>
+            dispatch(completeTodo(index))
+          } />
+        <Footer
+          filter={visibilityFilter}
+          onFilterChange={nextFilter =>
+            dispatch(setVisibilityFilter(nextFilter))
+          }
+          onUndo={() =>
+            dispatch(ActionCreators.undo())
+          }
+          onRedo={() =>
+            dispatch(ActionCreators.redo())
+          } />
```

Comment:
Alright I'll change it. I did it like that because it was in multiple lines in the example from the docs, I tried to keep my style consistent with that.